apps/rush-lib/src/api/EnvironmentConfiguration.ts

```diff
@@ -68,7 +68,25 @@ export const enum EnvironmentVariableNames {
    * This environment variable can be used to specify the `--target-folder` parameter
    * for the "rush deploy" command.
    */
-  RUSH_DEPLOY_TARGET_FOLDER = 'RUSH_DEPLOY_TARGET_FOLDER'
+  RUSH_DEPLOY_TARGET_FOLDER = 'RUSH_DEPLOY_TARGET_FOLDER',
+
+  /**
+   * Overrides the location of the `~/.rush` global folder where Rush stores temporary files.
+   *
+   * @remarks
+   *
+   * Most of the temporary files created by Rush are stored separately for each monorepo working folder,
+   * to avoid issues of concurrency and compatibility between tool versions.  However, a small set
+   * of files (e.g. installations of the `@microsoft/rush-lib` engine and the package manager) are stored
+   * in a global folder to speed up installations.  The default location is `~/.rush` on POSIX-like
+   * operating systems or `C:\Users\YourName` on Windows.
+   *
+   * Use `RUSH_GLOBAL_FOLDER` to specify a different folder path.  This is useful for example if a Windows
+   * group policy forbids executing scripts installed in a user's home directory.
+   *
+   * POSIX is a registered trademark of the Institute of Electrical and Electronic Engineers, Inc.
+   */
+  RUSH_GLOBAL_FOLDER = 'RUSH_GLOBAL_FOLDER'
```

Comment:
Should this be `RUSH_USER_FOLDER` since that's what it currently is?

Reply:
I don't think so.  The intention is for it to be "global" i.e. shared across many Git enlistments that build on the same machine.  The reason we chose a user's home directory is technical -- security considerations require global folders to be configured manually, whereas we want this to "just work" without any special setup.